<!DOCTYPE HTML PUBLIC "-//W3C//DTD HTML 3.2//EN">
<HTML>
 <HEAD>
   <TITLE> [mod_python] Difficulties with Windows NT/Apache/mod_python
   </TITLE>
   <LINK REL="Index" HREF="index.html" >
   <LINK REL="made" HREF="mailto:mod_python%40modpython.org?Subject=%5Bmod_python%5D%20Difficulties%20with%20Windows%20NT/Apache/mod_python&In-Reply-To=Pine.LNX.4.21.0006151030550.23457-100000%40lade.src.no">
   <META NAME="robots" CONTENT="index,nofollow">
   <META http-equiv="Content-Type" content="text/html; charset=us-ascii">
   <LINK REL="Previous"  HREF="007508.html">
   <LINK REL="Next"  HREF="007512.html">
 </HEAD>
 <BODY BGCOLOR="#ffffff">
<table border=0 width="100%"><tr><td valign="top">
   <H1>[mod_python] Difficulties with Windows NT/Apache/mod_python</H1>
    <B>Hr. Daniel Mikkelsen</B> 
    <A HREF="mailto:mod_python%40modpython.org?Subject=%5Bmod_python%5D%20Difficulties%20with%20Windows%20NT/Apache/mod_python&In-Reply-To=Pine.LNX.4.21.0006151030550.23457-100000%40lade.src.no"
       TITLE="[mod_python] Difficulties with Windows NT/Apache/mod_python">daniel at copyleft.no
       </A><BR>
    <I>Thu Jun 15 11:03:42 EST 2000</I>
    <P><UL>
        <LI>Previous message: <A HREF="007508.html">[mod_python] Difficulties with Windows NT/Apache/mod_python
</A></li>
        <LI>Next message: <A HREF="007512.html">[mod_python] python rewrite of mod_usertrack
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#7507">[ date ]</a>
              <a href="thread.html#7507">[ thread ]</a>
              <a href="subject.html#7507">[ subject ]</a>
              <a href="author.html#7507">[ author ]</a>
         </LI>
       </UL>
    <HR>  
<!--beginarticle-->
<PRE>&gt;<i> make_obcallback(): could not import mod_python.apache.
</I>&gt;<i> make_obcallback(): could not call init.
</I>&gt;<i> Apache/1.3.12 (Win32) mod_python/2.2 Python/1.5.2 running...
</I>
I got these messages when I copied the _files_ in the
lib/python/mod_python dir into site-packages, instead 
of copying the entire _directory_, by mistake. 

Anyway, it's a problem of your python interpreter not 
finding the mod_python modules. Use PYTHONPATH in the 
environment or PythonPath in the apache.conf. 

-- Daniel Mikkelsen, Copyleft Software



</PRE>
<!--endarticle-->
    <HR>
    <P><UL>
        <!--threads-->
	<LI>Previous message: <A HREF="007508.html">[mod_python] Difficulties with Windows NT/Apache/mod_python
</A></li>
	<LI>Next message: <A HREF="007512.html">[mod_python] python rewrite of mod_usertrack
</A></li>
         <LI> <B>Messages sorted by:</B> 
              <a href="date.html#7507">[ date ]</a>
              <a href="thread.html#7507">[ thread ]</a>
              <a href="subject.html#7507">[ subject ]</a>
              <a href="author.html#7507">[ author ]</a>
         </LI>
       </UL>

</td>
<td align="right" valign="top">
<script type="text/javascript"><!--
google_ad_client = "pub-9718360309690383";
google_ad_width = 120;
google_ad_height = 600;
google_ad_format = "120x600_as";
google_color_border = "CCCCCC";
google_color_bg = "FFFFFF";
google_color_link = "000000";
google_color_url = "666666";
google_color_text = "333333";
//--></script>
<script type="text/javascript"
  src="http://pagead2.googlesyndication.com/pagead/show_ads.js">
</script>
</td>
</table>

<hr>
<a href="http://mailman.modpython.org/mailman/listinfo/mod_python">More information about the Mod_python
mailing list</a><br>
</body></html>
